### PR TITLE
suite-nonsplit-cart-permission-group

### PR DESCRIPTION
**Pull request by himanshu-pal-metacube**
https://spryker.atlassian.net/browse/CC-16782

suite non split cart permission group

### CODE REVIEW

File: tests/api/suite_nonsplit/glue/cart_endpoints/cart_permssion_groups/positive.robot [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Get_cart_permission_groups_by_cart_id
+    [Setup]    Run Keywords    I get access token for the customer:    ${yves_user.email}
+    ...  AND    I set Headers:    Authorization=${token}
+    ...  AND    I send a POST request:    /carts    {"data": {"type": "carts","attributes": {"priceMode": "${mode.gross}","currency": "${currency.eur.code}","store": "${store.de}","name": "${test_cart_name}-${random}"}}}
+    ...  AND    Save value to a variable:    [data][id]    cart_id
+    ...  AND    Response status code should be:    201
+    ...  AND    I send a GET request:    /company-users
+    ...  AND    Response status code should be:    200
+    ...  AND    Save value to a variable:    [data][0][id]    company_user_id
+    ...  AND    I send a POST request:    /carts/${cart_id}/shared-carts    {"data": {"type": "shared-carts","attributes": {"idCompanyUser": "${company_user_id}","idCartPermissionGroup": "2"}}}
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/cart_endpoints/cart_permssion_groups/positive.robot:20]: 'idCartPermissionGroup' can be changed to a variable

Comment (himanshu-pal-metacube) [reply tests/api/suite_nonsplit/glue/cart_endpoints/cart_permssion_groups/positive.robot]: Added

**Review by Katerynaa-Lavryk — DISMISSED** [review]

**Comment by himanshu-pal-metacube** [comment]
We have gather all changes for the suite-nonsplit environment file in one PR. Once its approved by the reviewer we will merge it to master. It will reduced number of **conflicts** in Pending PR's.

File: resources/environments/environments_api_suite.json [hunk 2]
```diff
@@ -77,154 +97,406 @@
...
+        "brand_3": "Mauser",
+        "brand_4": "Sony",
+        "material_1": "aluminum",
+        "material_2": "metal",
+        "material_3": "chipboard",
+        "material_4": "Steel sheets, stove-enameled",
+        "delivery_1": "assembled",
+        "price_unit_1": "piece",
+        "color_1": "yellow",
+        "color_2": "Blue",
```
Comment (Katerynaa-Lavryk) [inline resources/environments/environments_api_suite.json:457]: I suggest 'Blue' should have same format as other colours and be written with lowercase - 'blue'.
Please re-check this and update if necessary

File: resources/environments/environments_api_suite.json [hunk 3]
```diff
@@ -237,27 +509,133 @@
...
         "cart_permission_group_id_2": "2",
-        "test_cart_name": "Test-cart"
-
+        "test_cart_name": "Test-cart",
+        "payment": {
+            "provider_name": "dummyMarketplacePayment",
+            "method_name": "invoice",
+            "selection_name": "dummyMarketplacePaymentInvoice",
+            "provider_name_1": "DummyPayment",
+            "method_name_1": "Invoice"
```
Comment (Katerynaa-Lavryk) [inline resources/environments/environments_api_suite.json:608]: Is it expected that "method_name" and "method_name_1" have same values but "method_name" is lowercase? Do we use both of them in tests or can any of them be redundant?

Comment (himanshu-pal-metacube) [reply resources/environments/environments_api_suite.json]: removed "method_name_1" from file

File: resources/environments/environments_api_suite.json (hunk 2 above)
Comment (himanshu-pal-metacube) [reply resources/environments/environments_api_suite.json:457]: I had re-checked the test-cases and result is same. No need to update.

**Review by Katerynaa-Lavryk — APPROVED** [review]

**Pull request merged**